src/Elewant/AppBundle/Repository/HerdRepository.php

```diff
@@ -74,7 +75,7 @@ public function search(string $searchString) : array
         return $query->getResult();
     }
 
-    public function findOneByShepherdId(ShepherdId $shepherdId) :? Herd
+    public function findOneByShepherdId(ShepherdId $shepherdId):? Herd
```

Comment:
This one is still not conform the rules; it should be `colon directly following the function arguments`, and `optional return value directly followed by return type`